packages/openapi3/src/schema-emitter-3-0.ts

```diff
@@ -149,7 +151,7 @@ export class OpenAPI3SchemaEmitter extends OpenAPI3SchemaEmitterBase<OpenAPI3Sch
     const variants = Array.from(union.variants.values());
     const literalVariantEnumByType: Record<string, any[]> = {};
     const ofType = getOneOf(program, union) ? "oneOf" : "anyOf";
-    const schemaMembers: { schema: any; type: Type | null }[] = [];
+    const schemaMembers: ScalarMember[] = [];
```

Comment:
This change crash https://cadlplayground.z22.web.core.windows.net/prs/6240/?c=bW9kZWwgUmFuZ2UgewogIGZyb206IElTT0RhdGUgfCBudWxsOwp9CgpzY2FsYXLJGmV4dGVuZHMgc3RyaW5nOwo%3D&e=%40typespec%2Fopenapi3&options=%7B%7D